.github/DISCUSSION_TEMPLATE/question.yml

```
@@ -0,0 +1,62 @@
+title: "Summary"
```

Last one, we should change this and hope people actually put a new title :)

```suggestion
title: "Question Title"
```